core/modules/system/css/system.theme.css

```diff
@@ -551,3 +551,10 @@ img.align-right {
   display: table;
   clear: both;
 }
+
+/**
+ * Labels & machine names.
+ */
+.machine-name {
+  font-size: smaller;
```

Comment:
I don't think we do this anywhere else in core. We should specify an actual size here.

Reply:
@jenlampton We do actually:
- https://github.com/backdrop/backdrop/blob/11619724ae5f80a1edaf957d367ce24c24aca4e4/core/themes/seven/css/seven.base.css#L93
- https://github.com/backdrop/backdrop/blob/11619724ae5f80a1edaf957d367ce24c24aca4e4/core/modules/system/css/system.maintenance.css#L9

Reply:
well alright then! :)